pages/index.vue

```diff
@@ -8,5 +8,12 @@
       Go to example
     </ULink>
   </div>
+</template>
 
-</template>
\ No newline at end of file
+<script lang="ts">
+import { fetchIntro, fetchImageUrl } from "~/api/wikipediaSource";
+const s : string = await fetchIntro("Jupiter");
```

Comment:
`await` not necessary here
```suggestion
const s : string = fetchIntro("Jupiter");
```

Reply:
Since `fetchIntro` is returning a `Promise`, we want it to be resolved before displaying it. The `await` keyword allows us to directly extract the `string` from the `Promise` once it is resolved.